extensions/2.0/Vendor/EXT_feature_metadata/examples/texture-metadata.gltf

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
Note that an example I'm working on will be very similar to this one but more fleshed out.

In time we might want to remove this file.

Reply:
@lilleyse once we've finalized `weather.gltf`, do we need `texture-metadata.gltf` and `vertex-metadata.gltf` anymore?

Reply:
Fine to remove `texture-metadata.gltf` and `vertex-metadata.gltf` once the other samples are finalized

Reply:
I removed texture-metadata, the other one I'll keep around for reference until I finish fixing `weather`